Makefile

```
@@ -25,8 +25,6 @@ clean:
 .PHONY: tag
 tag:
 	@git tag "${TAG}" || (echo "Tag ${TAG} already exists. If you want to retag, delete it manually and re-run this command" && exit 1)
-	@git-chglog --output CHANGELOG.md
-	@git commit -m 'Update CHANGELOG.md' -- CHANGELOG.md
 	@git tag -f "${TAG}"
```

This is not related with your changes, but I think this should be changed to be:
```suggestion
	@git tag -s -a -m "${TAG}"  "${TAG}"
```